app/views/users/_liquid.html.erb

```diff
@@ -15,7 +15,7 @@
     <h2><a href="<%= user.path %>" class="ltag__user__link"><%= user.name %></a><%= follow_button %></h2>
     <div class="ltag__user__summary">
       <a href="<%= user.path %>" class="ltag__user__link">
-        <%= user.summary %>
+        <%= user.summary_html&.html_safe %>
```

Comment:
Hi @bolariin, this is still not correct :) For users that don't have the summary_html this is going to generate an error. If you type nil.html_safe in your console you'll see.

You have to do the same thing everywhere: if there's a summary_html render that, otherwise render the summary.

Reply:
I'm fixing this now